packages/app/src/App.tsx

```diff
@@ -65,7 +65,7 @@ function App(props: Props) {
           flex={25}
           minSize={150}
         >
-          <Explorer selectedPath={selectedPath} onSelect={onSelectPath} />
+          <ExplorerTabs selectedPath={selectedPath} onSelect={onSelectPath} />
```

Comment:
What do you think of using a ui-less tab library, like https://headlessui.com/react/tabs? It would take care of the tab switching logic and of all the accessibility features for us.

The tabs code can still live in a separate component (to handle the `getSearchablePaths` logic and `searchText` state), but I would rename this component `Sidebar` (and the class of the parent reflex container `sidebarArea` maybe?), as when `getSearchablePaths` is not supported, we don't render any tabs.

Reply:
I tried to replicate the tab logic we had for the Vis but I wouldn't mind using something else.

Reply:
Meh, it can wait. I'm fine if you just do the renaming.